src/routes/docs/contributing/translating/+page.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Translating Files
+---
+
+Files uses Crowdin for translation management, which is a cloud-based SaaS that supports globalization of a variety of application platforms.
+
+## Getting started with Crowdin
+
+- Any browser that work on Windows with the latest version
+- Crowdin account
```

Comment:
Should we add a link so users can setup an account?

Reply:
Done